### DISCUSSION TITLE
Save time by adding Create Collection and New Folder button on a schema itself

### DISCUSSION THREAD

**Discussion by navanshu** [directus/directus#19734] [Feature Requests]
### Summary

These two icons can be added as shown in the diagram, they will only be visible when some one hovers on the schema (what ever it is called). They will save time of back and forth scrolling, dragging and dropping.

### Basic Example

![image](https://github.com/directus/directus/assets/7664426/b58145a5-dcdf-4d11-bddf-350d40abf6ab)

### Motivation

Main purpose is to improve the use experience of directus

### Detailed Design

You guys can figure it out 🤞🏽 

### Must Have:
- New schema (Collection) Icon
- New Folder Icon
- Show them only on hover

### Drawbacks

None

### Alternatives

Nothing else will be this accessible, or prevent the scroll position from changing when opening and closing a schema

### Adoption Strategy

No migration

**Comment by rijkvanzanten** [comment]
@benhaynes thoughts feelings concerns? Makes sense to me, but might clutter the rows a bit 🙂

**Reply by benhaynes** [reply]
I think these dedicatd buttons could work, but being aware of mobile, longer collection names, etc... we might want to add these new options to the "..." context menu:

* Add Collection Within
* Add Folder Within

Either way, this is a great addition!

**Reply by navanshu** [reply]
Even if the Data Model Part has layout options like one can view it as folders and then, the existing buttons on the topbar generate new collection or folder in the currently open folder can also provide same level of comfort, the active folder can be maintained in the URL or local storage.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
